techniques/general/G226.html

```diff
@@ -0,0 +1,51 @@
+   <p>Someone creating an instructional video demonstrating the features of an application prepares a script where what is being shown visually is reinforced through the narration, to arrive at an efficient and cost-effective means of making a more accessible video.</p>
+   <p>Several key strategies are followed to improve the video's narration. Each is described more fully in the following subsections:</p>
+   <ol>
+      <li>Describe any pertinent and meaningful text on the screen</li>
+      <li>Avoid saying only “this” or “here” to describe UI components</li>
+      <li>For better context, describe elements by sensory perceptions as well as by label</li>
+      <li>Fully describe sequences of action, including any dynamic content that appears</li>
+      <li>When a main page or dialog appears, say its title and describe its features</li>
+      <li>When using a mouse to show something (such as to hover, select, scroll, and open), say what you are doing</li>
+   </ol>
+   <h4>Describe any pertinent and meaningful text on the screen</h4>
+   <p>When referring to URLs, dialogs, labels, and headings, read out the text. Sometimes presenters (narrators) just highlight or point to text; speaking the visible text ensures this meaningful text is made accessible to everyone including blind users or those with low vision. When describing actions a user can do, be sure to specifically state the button names to improve the context (for example, "choose the green 'Go' button").</p>
+   <h4>Avoid saying only “this” or “here” to describe UI components</h4>
+   <p>This goes hand in hand with the first rule to announce text on the screen. When presenters (narrators) point out “this button” or say “you'll see this”, they are typically referring to a visual cue they are providing on the screen. Someone who can't see the screen lacks the context to understand what is being referenced. Replace or augment “this” and “here” with the labels/titles to provide context: “Choose the blue Save button”, “The Profile Settings dialog appears, with several options.”</p>
+   <h4>For better context, describe elements by sensory perceptions as well as by label</h4>
+   <p>Including position and other sensory qualities like color can really help some low vision users and users with cognitive disabilities. However, you will want to include other context, such as structural headings, in addition to position (which is usually not very helpful to blind user). For components with visible labels, always read out the label when referring to the component. Where a visible label is absent, but you are aware of other programmatic labeling that will be read by the screen reader (for example, the aria-label property or page regions), use that text. Also include placement and structure (headings) on the page (for example, “the red “Cancel” button at the bottom right of the dialog”, “Select the “online only” radio button in the Settings options”).</p>
```

Comment:
Maybe put the property name into a `code` element

```suggestion
   <p>Including position and other sensory qualities like color can really help some low vision users and users with cognitive disabilities. However, you will want to include other context, such as structural headings, in addition to position (which is usually not very helpful to blind user). For components with visible labels, always read out the label when referring to the component. Where a visible label is absent, but you are aware of other programmatic labeling that will be read by the screen reader (for example, the <code>aria-label</code> property or page regions), use that text. Also include placement and structure (headings) on the page (for example, “the red “Cancel” button at the bottom right of the dialog”, “Select the “online only” radio button in the Settings options”).</p>
```